cibuildwheel/macos.py

```diff
@@ -284,7 +287,9 @@ def build(options: Options, tmp_path: Path) -> None:
             env = before_all_options.environment.as_dictionary(prev_environment=os.environ)
             env.setdefault("MACOSX_DEPLOYMENT_TARGET", "10.9")
             before_all_prepared = prepare_command(
-                before_all_options.before_all, project=".", package=before_all_options.package_dir
+                before_all_options.before_all,
+                project=".",
+                package=before_all_options.package_dir,
```

Comment:
Would you mind to revert this change? Thanks.

Reply:
Be sure to avoid adding a trailing comma. That tells black you want it one per line.

Reply:
My mistake, fixed!